### DISCUSSION TITLE
posts/zelda_totk

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#20] [General]
# posts/zelda_totk

即使在十年前的老硬件上，也能创作出年度最佳游戏。真正令人激动的故事情节和极具创意的游戏机制远比图像质量的提升更为重要。

https://www.whexy.com/posts/zelda_totk

**Comment by whexy** [comment]
今天读了一篇公众号文章，得知《王国之泪》开发团队的平均年龄是55岁。

**Comment by TravisRoad** [comment]
个人感觉硬件包括技术发展都利好独立开发者，当大家不需要去做高级的hack，去抠索性能，大家就更能表达自己想表达的（现在游戏开发仍然是手工生产的劳动密集型行业，悲
